x/application/client/cli/tx_delegate_to_gateway.go

```diff
@@ -3,33 +3,42 @@ package cli
 import (
 	"strconv"
 
+	"pocket/x/application/types"
+
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
 	"github.com/spf13/cobra"
-	"pocket/x/application/types"
 )
 
 var _ = strconv.Itoa(0)
 
 func CmdDelegateToGateway() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "delegate-to-gateway",
-		Short: "Broadcast message delegate-to-gateway",
-		Args:  cobra.ExactArgs(0),
+		Use:   "delegate-to-gateway [gateway address]",
```

Comment:
Not sure if this is a NIT or not but I wonder if it's worth specifying that this is a bech32 address?

```suggestion
		Use:   "delegate-to-gateway [gateway bech32 address]",
```

Reply:
I think it's okay to avoid the `bech32` mention here. Might end up being too verbose